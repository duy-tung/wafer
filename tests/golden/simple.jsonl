{"id":"550e8400-e29b-41d4-a716-446655440000","source_file":"multiline.txt","chunk_index":0,"text":"This is a multi-line test file for golden file testing.\r\n\r\nIt contains multiple paragraphs and line breaks to test how the wafer CLI tool handles different text structures and formatting.\r\n\r\nThe third paragraph includes some special characters: !@#$%^\u0026*()_+-={}[]|;':\",./\u003c\u003e?\r\n\r\nThis ensures comprehensive testing of the text processing pipeline.","embedding":[0.1,0.2,0.3,0.4,0.5],"word_count":46,"created_at":"2024-01-15T10:30:45Z"}
{"id":"550e8400-e29b-41d4-a716-446655440000","source_file":"simple.txt","chunk_index":0,"text":"This is a simple test file for golden file testing. It contains exactly fifty words to test the chunking algorithm and ensure that the wafer CLI tool produces consistent, reproducible output for regression testing and validation purposes.","embedding":[0.1,0.2,0.3,0.4,0.5],"word_count":37,"created_at":"2024-01-15T10:30:45Z"}