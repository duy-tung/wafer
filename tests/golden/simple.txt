This is a simple test file for golden file testing. It contains exactly fifty words to test the chunking algorithm and ensure that the wafer CLI tool produces consistent, reproducible output for regression testing and validation purposes.
